### PR TITLE
fix(app): cal check: tweak copy, fix weird video asset, wrap tip rack names

### PR DESCRIPTION
**Pull request by b-cooper**
## overview / changelog

A couple of polish points for the calibration check flow, following up on the most recent design review.
 
 - the word "extremely" is removed from the copy on the introduction screen (this doesn't have to be _that_ intense
 - video asset for single channel pipette on right mount checking slot 1 replaced to remove mysterious vertical bars
 - title bar button is now "exit" instead of "back"
 - tip rack names in deck setup view now wrap

## review requests

- verify whole tip rack name is visible when viewing the deck setup screen in the calibration check flow
- Back button in title bar should be  "Exit"
- All video assets for xy and z checks should not have vertical black bars on right and left borders of video frame  

## risk assessment

low, small changes, still behind feature flag

### CODE REVIEW

**Review by sfoster1 — APPROVED** [review]
lgtm!

File: app/src/components/CheckCalibration/Introduction.js [hunk 1]
```diff
@@ -14,10 +14,8 @@ const INTRO_ROBOT_CALIBRATION_CHECK_BODY =
 const TIPRACK_REQS = 'For this process you will require:'
 const VIEW_TIPRACK_MEASUREMENTS = 'View measurements'
 const NOTE_HEADER = 'Please note: '
-const IMPORTANCE_MODIFIER = 'extremely'
-const NOTE_BODY_1 = "It's "
-const NOTE_BODY_2 =
-  ' important you perform this test using the Opentrons tips and tipracks specified above, as the robot determines accuracy based on the measurements of these tips.'
+const NOTE_BODY =
+  "It's important you perform this test using the Opentrons tips and tipracks specified above, as the robot determines accuracy based on the measurements of these tips."
 const CANCEL = 'Cancel'
```
Comment (shlokamin) [inline app/src/components/CheckCalibration/Introduction.js:19]: not explicitly introduced in this PR but looks like all the other const strings use single quotes

**Review by shlokamin — APPROVED** [review]
Looks good! small comment about single quotes

File: app/src/components/CheckCalibration/Introduction.js (hunk 1 above)
Comment (b-cooper) [reply app/src/components/CheckCalibration/Introduction.js:19]: Oh yeah, I made that one double quotes so I wouldn't have to escape the apostrophe in "It's".

**Pull request merged**